libPhoneNumber/NBMetadataHelper.m

```diff
@@ -66,73 +65,19 @@ + (NSDictionary *)phoneNumberDataMap {
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
       @autoreleasepool {
-          result = [self jsonObjectFromZippedDataWithBytes:kPhoneNumberMetaData
-                                          compressedLength:kPhoneNumberMetaDataCompressedLength
-                                            expandedLength:kPhoneNumberMetaDataExpandedLength];
-
-          // The jsonMap is large and held in memory.
-          // It's contents are deeply nested and very repetitive.
-          // We can greatly reduce the memory used by jsonMap
-          // by traversing its contents and de-duplicating repeated values.
-          result = [self deduplicateJsonMap:result];
+          NSString *path = [[NSBundle bundleForClass:NBMetadataHelper.class] pathForResource:@"phoneNumberMap" ofType:@"plist"];
+          NSData *fileContent = [NSData dataWithContentsOfFile:path];
+          if (fileContent != nil) {
```

Comment:
This should NSAssert if it's nil, without this map the whole library is broken.

Reply:
ok